### PR TITLE
Cshe/default route fix

### PR DESCRIPTION
**Pull request by ChengyuShe**
Removed the "ratemycse" part from the URL and fixed an issue that caused the default URL to not display the blue background on the home page.

### CODE REVIEW

**Review by twang7310 — COMMENTED** [review]
I think .vercel folder was added to gitignore by Jeremy, so it shouldn't be pushed to main.
Do this on your branch please:

git fetch origin jnguyen/add-db
git merge

And then push again

**Review by twang7310 — CHANGES_REQUESTED** [review]
I think .vercel folder was added to gitignore by Jeremy, so it shouldn't be pushed to main.
Do this on your branch please:

git fetch origin jnguyen/add-db
git merge

And then push again

File: app/src/App/App.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {BrowserRouter as Router, Route, Routes} from 'react-router-dom';
 import {InnerPage, HomeLayout} from '../Homepage/Homepage';
-import {GetDefaultRoute} from '../Helpers/utils';
+// import {GetDefaultRoute} from '../Helpers/utils';
```
Comment (thaowii) [inline app/src/App/App.tsx:3]: Don't leave commented out code

**Review by thaowii — APPROVED** [review]

**Review by thaowii — CHANGES_REQUESTED** [review]
Remove .vercel then push

**Review by celestinebuendia — CHANGES_REQUESTED** [review]
Change the tests you are failing (in App.test.tsx, the routing tests) so that they no longer contain 'rate-my-cse/' as a prefix

File: app/.vercel/README.txt
Comment (celestinebuendia) [inline app/.vercel/README.txt:1]: Delete this .vercel folder then commit

**Review by celestinebuendia — CHANGES_REQUESTED** [review]
Remove the .vercel folder

**Review by celestinebuendia — APPROVED** [review]
Approving because you changed the tests and Thao's review asks for you to remove .vercel.

**Review by thaowii — APPROVED** [review]

**Review by twang7310 — APPROVED** [review]

**Pull request merged**